src/main/java/frc/robot/commands/ColorWheelWithJoystick.java

```diff
@@ -0,0 +1,123 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Const;
+import frc.robot.subsystems.Elevator;
+import edu.wpi.first.wpilibj.DriverStation;
+import com.revrobotics.ColorMatch;
+import com.revrobotics.ColorMatchResult;
+import com.revrobotics.ColorSensorV3;
+
+
+public class ColorWheelWithJoystick extends CommandBase {
+    private final I2C.Port i2cPort = I2C.Port.kOnboard;
+    private final ColorSensorV3 colorSensor = new ColorSensorV3(i2cPort);
```

Comment:
If I'm being picky, I'd say don't make a separate variable for this, since you only use it once.